protocol/linkpreview_unfurler_opengraph.go

```diff
@@ -89,13 +128,15 @@ func fetchThumbnail(logger *zap.Logger, httpClient *http.Client, url string) (co
 		return thumbnail, fmt.Errorf("could not fetch thumbnail url='%s': %w", url, err)
 	}
 
-	width, height, err := images.GetImageDimensions(imgBytes)
-	if err != nil {
-		return thumbnail, fmt.Errorf("could not get image dimensions url='%s': %w", url, err)
+	// If an image is an SVG, We cannot get it's dimensions
+	if !images.IsSVG(imgBytes) {
+		width, height, err := images.GetImageDimensions(imgBytes)
```

Comment:
What if the image is an `ico`? Would you get the dimensions back? If so, we're not storing them in the protocol (only `favicon_payload`), so is there any need to get `ico`'s dimensions?

Reply:
The thing is, attempting to fetch SVGs dimensions breaks the entire thing
Because SVGs are markup and they need to be rendered in order to fetch their dimension
But you're right, I should omit calculating dimensions for favicons completely.